spec/models/wotsit_spec.rb

```diff
@@ -11,6 +11,40 @@
     expect(reified.updated_at).not_to(be_nil)
   end
 
+  context 'with rails 7.1 attr_readonly' do
+    context 'with rails 7.1 config, default is true' do
```

Comment:
If I am reading it correctly, this is the behavior pre-7.1. I think that means the context names can be generalized.

Reply:
Thank you for the feedback, I've updated the tests.